File: Methods-Python/selection_sort.py
def selection_sort(list1):

    for i in range(0, len(list1)):
        min1 = 1
        for j in range(i + 1, len(list1)):
            if list1[j] < list1[min1]:
                min1 = j

        tmp = list1[i]
        list1[i] = list1[min1]
        list1[min1] = tmp

    return list1

File: Methods/Methods-C++/selection_sort.cpp
#include <iostream>
#include <math.h>

int* selection_sort(int list[], int n) {
    int i;
    int j;
    int min;
    for (i = 0; i < n - 1; i++) {
        min = i;
        for (j = i + 1; j < n; j++) {
            if (list[j] < list[min]) {
                min = j;
            }
        }
        int tmp = list[i];
        list[i] = list[min];
        list[min] = tmp;
    }
    return list;
}